index.html

```diff
@@ -0,0 +1,201 @@
+                        <li class="tag">css</li>
+                        <li class="tag">javascript</li>
+                    </ul>
+                    <div>
+                        <button class="btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+    
+        </section>
+    
+        <section id="about-myself" class="presentation">
+            <h1>About <br>Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                me.</p>
+            <a href="#">Let's connect</a>
+            <ul>
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/Linkedin icon.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angelist"></a></li>
+            </ul>
+            <div id="resume">
+                <button class="btn">Get my resume</button>
+            </div>
+            <div class="skill-set">
+                <div id="language">
+                    <h3>Languages</h3>
+                    <img src="images/down-arrow.png" height="12px"/>
+                </div>
+    
+                <ul id="language-list">
+                    <li>
+                        <img src="images/js.png" />
+                   </li>
+                    <li>
+                        <img src="images/html.png" />
+                    </li>
+                    <li>
+                        <img src="images/css.png" />
+                    </li>
+                </ul>
+        
+                <div id="language" class="border-bot">
+                    <h3>Frameworks</h3>
+                    <img src="images/right-arrow.png" height="12px"/>
+                </div>
+                <div id="language" class="border-bot">
+                    <h3>Skills</h3>
+                    <img src="images/right-arrow.png" height="12px"/>
+                </div>
+            </div>
+            
+        </section>
+
+        <footer>
+            <div id="contact-form">
+                <h1>Contact me</h1>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project
+                    that
+                    needs coding. I’d love to help with it </p>
+                <form action="https://formspree.io/f/maykokbe" method="post" class="form-container">
+                    <input type="text" name="fullname" maxlength="30" placeholder="Enter your fullname"/>
+                    <input type="email" name="email" placeholder="Enter your email"/>
```

Comment:
- [ ] Good job on your form implementation 👍🏽. Kindly note that all input fields should have the` required` attribute to activate the validation on submission so as to prevent empty data from getting submitted.